jaq-core/src/time.rs

```diff
@@ -30,24 +29,32 @@ pub fn to_iso8601(v: &Val) -> Result<String, Error> {
         })
         .encode();
 
-    let fai1 = |e| Error::str(format!("cannot format {v} as ISO-8601 timestamp: {e}"));
-    let fai2 = |e| Error::str(format!("cannot format {v} as ISO-8601 timestamp: {e}"));
+    let fail1 = |e| {
+        Error::str(&format_args!(
+            "cannot format {v} as ISO-8601 timestamp: {e}"
+        ))
+    };
+    let fail2 = |e| {
+        Error::str(&format_args!(
+            "cannot format {v} as ISO-8601 timestamp: {e}"
+        ))
+    };
```

Comment:
This is much too long. Perhaps only output `format_args` instead of `Error::str`?

Reply:
This is fixed by removing the reference in https://github.com/01mf02/jaq/pull/124/commits/4828b4f84b4cf0b530909c3e51d1a3afdd999657